### PR TITLE
docs: add a miniature article about observability

### PR DESCRIPTION
**Pull request by freben**
It isn't much, but it needed to be put somewhere :)

### CODE REVIEW

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4339-dac8sfi2nanerisomcftmrkhfjfk8cuo.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/601827d298552d3aa553c434

File: docs/plugins/observability.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+There is a basic Google Analytics integration built into Backstage. You can
+enable it by adding the following to your app configuration:
+
+```yaml
+app:
+  googleAnalyticsTrackingId: UA-000000-0
+```
+
+Replace the tracking ID with your own.
```
Comment (adamdmharvey) [inline docs/plugins/observability.md:21]: Maybe also add...

```suggestion
Replace the tracking ID with your own.

For information, 
[learn more about Google Analytics](https://marketingplatform.google.com/about/analytics/).
```

**Review by adamdmharvey — APPROVED** [review]
Very nice; short and sweet 🎉

File: docs/plugins/observability.md (hunk 1 above)
Comment (freben) [reply docs/plugins/observability.md:21]: Sounds good!

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4339-dac8sfi2nanerisomcftmrkhfjfk8cuo.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/601827d298552d3aa553c434

**Review by adamdmharvey — APPROVED** [review]
👍

**Comment by freben** [comment]
added sections on health checks and logging while i was at it

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4339-dac8sfi2nanerisomcftmrkhfjfk8cuo.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/601827d298552d3aa553c434

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4339-dac8sfi2nanerisomcftmrkhfjfk8cuo.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/601827d298552d3aa553c434

**Review by benjdlambert — APPROVED** [review]

**Review by Rugvip — APPROVED** [review]

**Pull request merged**